src/frontend.md

```diff
@@ -5,31 +5,56 @@
 ```
 src/
   components/ (can be dumb and smart)
-    bankAccount/
+    [All components should be placed in some entity folder or shared folder]
+    bankAccount/ (entity, module, .etc)
       BankAccountTable/
         [no nested directories]
+        [avoid index.ts]
         BankAccountTable.ts
         BankAccountTable.test.ts
         BankAccountTableBadge.ts (child component)
         BankAccountTable.styled.ts
+        BankAccountTable.hook.ts (ui hook used only in the current component)
+        icon.svg (the icon used only in the current component)
+        [avoid index.ts]
+        [other files should be placed in queries or services]
     shared/
       SomeNonEntityComponent/
+      media/ (shared icons)
+        [no nested directories]
+        icon.svg
   pages/
     bankAccount/
       BankAccountList/
+        [no nested directories]
+
         BankAccountList.page.ts
         BankAccountList.page.test.ts
+        [components for the page should be placed in components folder]
+        [other files should be placed in queries or services]
   queries/
-    bankAccount.query.ts
+    bankAccount.query.ts (query hook)
```

Comment:
```suggestion
    bankAccount.query.ts (query hooks)
```

Reply:
We consider that a hook is a query hook if it uses `react-query` and `sdk`. It might also include mutations, some data normalisation etc. We should avoid hooks that include both a query and some other business logic.